Regular rebase at 2022 July

cyxlily opened the pull request:
Signed-off-by: Cui, Yuxin <yuxin.cui@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4358

sysopenci commented:
Jira issue not in ['bug','improvement'], make sure jira issue is of type bug or improvement.

Commit:
Regular rebase at 2022 July

Update trusty from S to T.

Manifest tag: 2022-July-release
Tracked-On: OAM-103621
Signed-off-by: Cui, Yuxin <yuxin.cui@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4407

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4408

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4409

Done by "Cui, Yuxin" <yuxin.cui@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/2523

Done by "Cui, Yuxin" <yuxin.cui@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Joshi, Shweta" <shweta.joshi@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/2938
